setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-install_requires=['esupy @ git+https://github.com/USEPA/esupy.git@develop#egg=esupy',
+install_requires=['esupy @ git+https://github.com/USEPA/esupy.git@urban_rural_classify#egg=esupy',
```

Comment:
after we pull in https://github.com/USEPA/esupy/pull/33 let's point this back to develop on this PR